### PR TITLE
[MI-2203] Used a shorter commit id for notification post

### PR DESCRIPTION
**Pull request by raghavaggarwal2308**

### CODE REVIEW

File: server/plugin/api.go [hunk 1]
```diff
@@ -446,7 +446,7 @@ func (p *Plugin) handleSubscriptionNotifications(w http.ResponseWriter, r *http.
 				commits = ""
 			}
 
-			commits += fmt.Sprintf("\n[%s](%s) : **%s**", body.Resource.Commits[i].CommitID, body.Resource.Commits[i].URL, body.Resource.Commits[i].Comment)
+			commits += fmt.Sprintf("\n[%s](%s) : **%s**", body.Resource.Commits[i].CommitID[0:8], body.Resource.Commits[i].URL, body.Resource.Commits[i].Comment)
```
Comment (ayusht2810) [inline server/plugin/api.go:449]: are we always sure about its length?

Comment (raghavaggarwal2308) [reply server/plugin/api.go:449]: Yes

**Review by ayusht2810 — APPROVED** [review]

File: server/plugin/api.go (hunk 1 above)
Comment (avas27JTG) [inline server/plugin/api.go:449]: @raghavaggarwal2308 is this just about truncating the commit-id up to this limit or something else, as on slack screenshot or `git log --oneline` I think we get a different ID?

Comment (raghavaggarwal2308) [reply server/plugin/api.go:449]: Yes, its just about truncation the commit-id. The smaller commit IDs are also unique in the particular repo, so we can use them to uniquely identify a commit
https://support.configura.com/hc/en-us/articles/360052644053-Are-Git-Commit-id-s-Truncated-in-CET-Developer-

Comment (avas27JTG) [reply server/plugin/api.go:449]: Okay

**Review by avas27JTG — APPROVED** [review]

**Pull request merged**